latch_cli/nextflow/types.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class NextflowProcessExecutor(Enum):
+    local = "local"
```

Comment:
why is this an option?

Reply:
if someone want to execute a workflow using conda instead of docker or just execute locally it should still work
itll still generate the graph and stuff it just wont launch pods.

Reply:
my point was in this case why even use latch and not just run it locally

i think in this case we're giving people too much choice and causing potential confusion